search/api_test.py

```diff
@@ -191,6 +214,83 @@ def test_index_users(self):
         index_users(User.objects.iterator(), chunk_size=4)
         assert_search(search(), list(User.objects.all()))
 
+    def test_add_certificate(self):
```

Comment:
what do you think about defining a list of cached model factories (`CachedFactories = [CachedCertificateFactory, CachedEnrollmentFactory]`) and looping through those to perform these add/update/delete tests? this can potentially be done as part of the refactor story ( #743 ) if we ever get around to it. this is just a lot of repetition and it seems to make sense to treat these cases in a more generic way